### DISCUSSION TITLE
How to use nc_py_api in app skeleton?

### DISCUSSION THREAD

**Discussion by Lucy781228** [cloud-py-api/nc_py_api#242] [Q&A]
I have created my own app with the app skeleton: [https://apps.nextcloud.com/developer/apps/generate](url). Then I just need to move that directory into /apps. Now I want to create another app with Python code. Although I have read this link: [https://cloud-py-api.github.io/nc_py_api/](url) about nc_py_api, I still have no idea what my app structure is because the sample skeleton does not have a Python directory.
Also, does nc_py_api support Pandas and Streamlit?

**Comment by bigcat88** [comment]
this is a skeleton repository for python:

https://github.com/cloud-py-api/app-skeleton-python

this is an example with UI: https://github.com/cloud-py-api/ui_example

I do not know if "Streamlit" will work, probably will not, until some modifications, as ExApp can have UI only through AppAPI Proxy.

We will try write a small guide how to connect UI with concrete example of application that was not developed for Nextcloud from start, in next 2-4 weeks..
